components/BottomSheet/src/private/MDCDraggableView.m

```diff
@@ -106,4 +106,12 @@ - (BOOL)gestureRecognizer:(__unused UIGestureRecognizer *)recognizer
   return NO;
 }
 
+// Disable pan gesture on UIControl
+- (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldReceiveTouch:(UITouch *)touch {
+    
+    if ([touch.view isKindOfClass:[UIControl class]]) {
+        return NO;
+    }
+    return YES;
```

Comment:
Google style guide wants code to have 100 character per line
http://google.github.io/styleguide/objcguide.html#line-length

also indentation are 2 spaces.

This can all be fixed by `git clang-format $(git merge-base origin/develop HEAD)`